__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const app = require("../app");
+const db = require("../db/connection");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("undefined paths", () => {
+  test("404: Path not Found", () => {
+    return request(app)
+      .get("/beApi")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Path not Found!");
+      });
+  });
+});
+
+describe("GET /api/categories", () => {
+  test("200: retrieves categories", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+
+      .then(({ body: { categories } }) => {
+        expect(categories).toBeInstanceOf(Array);
```

Comment:
it's not inherently bad to include this line checking for instance of Array but as you're using a forEach and also checking length it's not adding anything in this case